conf/modules/generate_clinical_set.config

```diff
@@ -50,10 +50,16 @@ process {
 process {
     withName: '.*:GENERATE_CLINICAL_SET_MT:ENSEMBLVEP_FILTERVEP' {
         ext.when   = !params.skip_vep_filter
-        ext.prefix = { "${meta.id}_mt_${meta.set}" }
+        ext.prefix = { "${meta.id}_mt_filtervep_${meta.set}" }
         ext.args   = { "--filter \"HGNC_ID in ${feature_file}\"" }
     }
 
+    withName: '.*:GENERATE_CLINICAL_SET_MT:BCFTOOLS_FILTER' {
+        ext.when   = !params.skip_vep_filter
+        ext.prefix = { "${meta.id}_mt_${meta.set}" }
+        ext.args   = { "-Oz -i 'AF>0.05'" }
```

Comment:
Could be good to add a line in the documentation saying that the default is not to output variants with a VAF below 5%.

Reply:
Done! 👍🏻